resources/assets/components/pages/ShowSubmissionPage/RecommendedCampaignsGallery.js

```diff
@@ -28,11 +33,21 @@ const SCHOLARSHIP_CAMPAIGNS_QUERY = gql`
   ${scholarshipCardFragment}
 `;
 
-const RecommendedCampaignsGallery = () => (
-  <Query query={SCHOLARSHIP_CAMPAIGNS_QUERY}>
+const RecommendedCampaignsGallery = ({ variables }) => (
+  <Query
+    query={SCHOLARSHIP_CAMPAIGNS_QUERY}
+    variables={{
+      excludeIds: [
+        ...(variables.excludeIds || []),
+        ...siteConfig('hide_campaign_ids', []).map(id => Number(id)),
+      ],
+    }}
+  >
     {result => (
       <GalleryBlock
-        blocks={result.campaigns.edges.map(edge => edge.node.campaignWebsite)}
+        blocks={(get(result, 'campaigns.edges') || []).map(
```

Comment:
Nit - any reason we can't specify the empty array as the default value? I think it'd be a slight improvement for readability...  
```suggestion
        blocks={get(result, 'campaigns.edges', []).map(
```

Reply:
Surprisingly, (Or I suppose this actually makes sense), `get` will not return the default value if the object's value is assigned to `null` (which is the edge case here when GraphQL returns `null` results instead of the empty array we'd expect).

```js
> const lodash = require('lodash')
undefined
> lodash.get({}, 'nothing', [])
[]
> lodash.get({nothing: null}, 'nothing', [])
null
```